exercises/crypto-square/canonical-data.json

```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
-      "description": "empty plaintext results in an empty ciphertext",
-      "property": "ciphertext",
+      "description": "empty plainText results in an empty cipherText",
```

Comment:
I assume you deliberately chose to use the property names, but I think it would maybe be better to use `plain text` and `cipher text` here.

Reply:
fixed descriptions to not use property names in text.